### ISSUE TITLE
messages.getHistory

### ISSUE THREAD

**Issue by shabanovtg** [feathers-studio/telegraf-docs#20]
How I can get history messages, when I added bot to channel?

**Comment by MKRhere** [comment]
No, you cannot. This method is from the MTProto API (for user accounts) and not the Bot API (https://core.telegram.org/bots/api). Bots cannot access get message history. They can only listen for new messages.
